docs/docs/developers/getting_started/quickstart.md

```diff
@@ -15,11 +15,14 @@ In this guide, you will
 
 - Node.js >= v18 (recommend installing with [nvm](https://github.com/nvm-sh/nvm))
 
-## Install Docker
+## Install Docker daemon
```

Comment:
you're still installing docker, why include the word daemon?

Reply:
See docker instructions. Docker funnels people to think Docker = Docker Desktop. We explicitly want the daemon.
If they have speed issues running aztec/noir commands, we specifically wan't devs to close Docker desktop, but have the daemon running.